src/serializers/file/single.js

```diff
@@ -34,6 +46,7 @@ module.exports = async (instance, includes = []) => {
     "created_at",
     "updated_at",
     "download_url",
+    "imgix_url",
```

Comment:
@maheshmuttinti We can't include imgix_url for all instances. We should first check if env has IMGIX_URL and only then call setImgixUrl function, and also including in attributes should also be conditional.